temporian/core/operators/test/test_unary.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+import numpy as np
+from absl.testing import absltest
+
+from temporian.implementation.numpy.data.io import event_set
+from temporian.test.utils import assertOperatorResult, f64
+
+
+class UnaryTest(absltest.TestCase):
+    """Test numpy implementation of all unary operators"""
+
+    def test_invert_boolean(self) -> None:
+        """Test inversion of boolean features"""
+        evset = event_set(
+            timestamps=[0, 1, 2, 1, 1, 2, 0, 1],
+            features={
+                "store_id": [1, 1, 1, 1, 2, 2, 2, 2],
+                "product_id": [1, 1, 1, 2, 2, 2, 3, 3],
+                "boolean_1": [
+                    True,
+                    True,
+                    False,
+                    False,
+                    False,
+                    True,
+                    False,
+                    True,
+                ],
+                "boolean_2": [
+                    True,
+                    False,
+                    True,
+                    False,
+                    True,
+                    False,
+                    False,
+                    True,
+                ],
+            },
+            indexes=["store_id", "product_id"],
+        )
```

Comment:
remove feature names, make much shorter?

Reply:
You're right, this is unnecessarily long 😬